js/index.js

```diff
@@ -0,0 +1 @@
+// const projectName = 'technical-docs-page';
```

Comment:
App name was declared but not used and by now it is commented

Reply:
kindly for example here you could say please remove the comments or the code not used to make your code clean and readable.